code/test_odbiornik.ino

```diff
@@ -0,0 +1,57 @@
+#include <SoftwareSerial.h>
+ 
+SoftwareSerial mySerial(2, 3);
+
+unsigned long startTime = 0;
+
+
+void setup(){
+   Serial.begin(9600);
+   mySerial.begin(9600);
+   pinMode(4, INPUT);
+   while(digitalRead(4) == LOW);
+   startTime = millis();
+}
+int milisecond = 0;
+int second = 0;
+int minute = 0;
+int hour = 0;
+void denis(){
+  if (mySerial.available() > 0){
+    Serial.print(mySerial.readString());
+    Serial.println("/");
+    Serial.println(showTime());
+  }
+}
+void calculateTime(){
```

Comment:
To samo dzieje się w klasie TheTime z innego pliku. Czemu nie zrobić po prostu obiektu tej klasy i z tamtąd tego używać?